docs/variables/accountage.md

```diff
@@ -0,0 +1,33 @@
+---
+id: accountage
+---
+
+# $(accountage)
+
+Returns the time that has passed since a user created their **Twitch account**.
+
+#### Parameters
+
+This variable takes **one** *optional* parameter that is a Twitch username of who to fetch the account age of. Defaults to the sender's username if not provided.
+
+#### Example Output
+
+* `$(accountage)`
+
+    ```
+    Fossabot created their account 3 years, 6 months, 6 days and 13 hours ago
```

Comment:
The indentation is to make the code block belong to the same list item instead of having it break the list apart.

For some reason there only seems to be an indentation rule for ordered lists [**here**](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md029---ordered-list-item-prefix) and even that does not seem to be enforced even if specifically enabled. 🤷

There might be a setting somewhere, but I gave up after looking for an hour.

Reply:
Sounds good, I like the look of it, maybe later could add to a `README.md` about code style 👍